airflow/contrib/hooks/gcp_pubsub_hook.py

```diff
@@ -33,12 +36,13 @@ def _format_topic(project, topic):
     return 'projects/{}/topics/{}'.format(project, topic)
 
 
-class PubSubException(Exception):
-    pass
+# Alias for Exception
+PubSubException = Exception
```

Comment:
It seems to me that it changes the behavior of the code. What error is it related to?

Reply:
Unnecessary pass statement but this could be solved by leaving a docstring instead of bare `pass`.